docs/getting-started.jade

```diff
@@ -6,367 +6,182 @@ include ../_includes/utils
+    /** Map relative paths to URLs. */
+    const map: any = {
+      'angular2-google-maps': 'vendor/angular2-google-maps'
+    };
+
+    /** User packages configuration. */
+    const packages: any = {
+      'angular2-google-maps/core': {
+        defaultExtension: 'js',
+        main: 'index.js' // you can also use core.umd.js here, if you want faster loads
+      }
+    };
     ```
 
 :marked
-  ### tsconfig.json
-  To configure the TypeScript compiler for Angular 2 needs, we have to add the following **tsconfig.json** to our project folder.
+  ### Update angular-cli-build.js
+  Open the `angular-cli-build.js` located in the root folder.
 
-.code-highlight
-  :marked
-    ```markup
-    {
-      "compilerOptions": {
-        "target": "es5",
-        "module": "commonjs",
-        "moduleResolution": "node",
-        "sourceMap": true,
-        "emitDecoratorMetadata": true,
-        "experimentalDecorators": true,
-        "removeComments": false,
-        "noImplicitAny": false
-      },
-      "exclude": [
-        "node_modules",
-        "typings/main",
-        "typings/main.d.ts"
-      ]
-    }
-    ```
-    
-:marked
-  ### systemjs.config.js
-  Next, we have to configure SystemJS, which loads our JavaScript modules. Please add the following **systemjs.config.js** to the project folder.
+  Include angular2-gogole-maps in the `vendorNpmFiles` array. **Don't delete existing array entries.**
 
 .code-highlight
   :marked
-    ```markup
-    (function(global) {
-      // map tells the System loader where to look for things
-      var map = {
-        'app':                        'app', // 'dist',
-        'rxjs':                       'node_modules/rxjs',
-        '@angular':                   'node_modules/@angular'
-      };
-      // packages tells the System loader how to load when no filename and/or no extension
-      var packages = {
-        'app':                        { main: 'main.js',  defaultExtension: 'js' },
-        'rxjs':                       { defaultExtension: 'js' }
-      };
-      var packageNames = [
-        '@angular/common',
-        '@angular/compiler',
-        '@angular/core',
-        '@angular/http',
-        '@angular/platform-browser',
-        '@angular/platform-browser-dynamic',
-        '@angular/router',
-        '@angular/router-deprecated',
-        '@angular/testing',
-        '@angular/upgrade',
-      ];
-      // add package entries for angular packages in the form '@angular/common': { main: 'index.js', defaultExtension: 'js' }
-      packageNames.forEach(function(pkgName) {
-        packages[pkgName] = { main: 'index.js', defaultExtension: 'js' };
+    ```typescript
+    var Angular2App = require('angular-cli/lib/broccoli/angular2-app');
+
+    module.exports = function(defaults) {
+      return new Angular2App(defaults, {
+        vendorNpmFiles: [
+          ...
+          // add the following two lines:
+          'angular2-google-maps/**/*.js',
```

Comment:
@SebastianM This can be simplified to `'angular2-google-maps/**/*.+(js|js.map)'`